src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -8,66 +8,66 @@ public class Exercise extends ExerciseBase {
 
 
     // 1. Change the value of the member below to be the result of adding numOne and numTwo together
-    public int numOnePlusTwo = 0;
+    public int numOnePlusTwo = numOne + numTwo;
 
 
     // 2. Change the value of the member below to be the result of multiplying numThree by numTwo
-    public int numThreeTimesNumTwo = 0;
+    public int numThreeTimesNumTwo = numThree*numTwo;
 
 
     // 3. Change the value of the member below to be the result of dividing numThree by numOne
-    public int numThreeDividedByNumOne = 0;
+    public int numThreeDividedByNumOne = numThree/numOne;
 
 
     // 4. Change the value of the member below to be the result of subtracting numOne from numThree
-    public int numThreeMinusNumOne = 0;
+    public int numThreeMinusNumOne = numThree - numOne;
 
 
     // 5. Change the value of the member below to be the sum of numOne, numTwo and numThree
-    public int sum = 0;
+    public int sum = numOne + numTwo + numThree;
 
 
     // 6. Change the value of the member below to be the sum of numOne, numTwo and numThree divided by numOne
-    public int numBytes = 0;
+    public int numBytes = sum/numOne;
 
+    public String alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
 
     // 7. Create a public char member named lastLetter containing the last letter of the English alphabet
-
+    public char lastLetter = alphabet.charAt(alphabet.length() - 1);
```

Comment:
Often with coding the simplest approach is the best approach. What could you have done here to make this more simple?